### PR TITLE
Feature/no ref/projectcard

### PR DESCRIPTION
**Pull request by lushi-liu**
- Need to decide how to get the image set up and how to present them

Screenshot 2023-09-11 010021

### CODE REVIEW

File: portfolio-jsm/src/components/ProjectCard.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      <div className="flex flex-col md:flex-row">
+        <div className="ml-6 mt-12 flex flex-col md:ml-24 md:mt-20 md:w-[50%]">
+          <h1 className="text-32 font-extrabold md:text-48">{title}</h1>
+          <div className="mt-3 flex flex-row">
+            <Button
+              buttonColor="bg-[#FFFFFF33] rounded-[6px] py-3"
+              textColor="text-white-900 font-semibold"
+              hPadding="px-3"
+              width={""}
+              height={""}
```
Comment (brandonetter) [inline portfolio-jsm/src/components/ProjectCard.tsx]: Try making the props optional in the Button interface so you don't need to include things you're not using. You're already doing so with some of the props.

File: portfolio-jsm/src/components/ProjectCard.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+              onClick={() => {}}
+              text="React.JS, Next.js"
+            />
+            <div className="ml-4">
+              <Button
+                buttonColor="bg-[#FFFFFF33] rounded-[6px] py-3"
+                textColor="text-white-900 font-semibold"
+                hPadding="px-3"
+                width={""}
+                height={""}
```
Comment (brandonetter) [inline portfolio-jsm/src/components/ProjectCard.tsx]: Same here

File: portfolio-jsm/src/components/NavBar.tsx [hunk 3]
```diff
@@ -72,19 +72,74 @@ const NavBar = () => {
...
-          width={24}
-          height={24}
-        />
+      <div className="mt-5 flex flex-row md:hidden">
+        <button onClick={toggleTheme} className="mr-4">
+          {theme === "dark" ? (
+            <Image src="./moon.svg" alt="" width={20} height={20} />
+          ) : (
+            <Image src="./sun.svg" alt="" width={20} height={20} />
+          )}
```
Comment (brandonetter) [inline portfolio-jsm/src/components/NavBar.tsx]: This is fine. But you can just use a ternary on the image src itself.

File: portfolio-jsm/src/components/Input.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  return (
+    <input
+      type={type}
+      onClick={
+        type === "Time"
+          ? (e) => {
+              e.currentTarget.showPicker();
+            }
+          : undefined
+      }
```
Comment (brandonetter) [inline portfolio-jsm/src/components/Input.tsx]:
```suggestion
  let onClick;
  
  if (type === "Time") {
    onClick = (e: React.MouseEvent<HTMLInputElement>) => {
      e.currentTarget.showPicker();
    };
  }

  return (
    <input
      type={type}
      onClick={onClick}
      placeholder={placeholder}
      className={`block rounded-[10px] ${inputColor} px-4 py-2`}
      style={inputStyle}
      onChange={onChange}
    />
  );
```

Not a huge fan of this ternary. Consider this

**Review by brandonetter — CHANGES_REQUESTED** [review]
As far as your comment about images for those display sections- look into "overflow-hidden" and positioning.

File: portfolio-jsm/src/components/Input.tsx (hunk 4 above)
Comment (lushi-liu) [reply portfolio-jsm/src/components/Input.tsx]: I realized I'm not going to use the time picker property so I removed it

File: portfolio-jsm/src/components/NavBar.tsx [hunk 5]
```diff
@@ -64,11 +64,12 @@ const NavBar = () => {
...
         <div className="mr-12 dark:text-white-800">|</div>
         <button onClick={toggleTheme} className="mr-12">
-          {theme === "dark" ? (
-            <Image src="./moon.svg" alt="" width={20} height={20} />
-          ) : (
-            <Image src="./sun.svg" alt="" width={20} height={20} />
-          )}
+          <Image
+            src={theme === "dark" ? "./moon.svg" : "./sun.svg"}
+            alt=""
```
Comment (brandonetter) [inline portfolio-jsm/src/components/NavBar.tsx:69]: Let's add an 'alt' here for accessibility

**Review by brandonetter — CHANGES_REQUESTED** [review]
Good work so far. One small change for accessibility

File: portfolio-jsm/src/components/NavBar.tsx (hunk 5 above)
Comment (brandonetter) [reply portfolio-jsm/src/components/NavBar.tsx:69]: Once that's fixed you can merge this

**Pull request merged**